lib/web/apiserver.go

```diff
@@ -521,23 +521,23 @@ func (h *Handler) bindDefaultEndpoints() {
 	// endpoint returns the default authentication method and configuration that
 	// the server supports. the /webapi/ping/:connector endpoint can be used to
 	// query the authentication configuration for a specific connector.
-	h.GET("/webapi/ping", httplib.MakeHandler(h.ping))
-	h.GET("/webapi/ping/:connector", httplib.MakeHandler(h.pingWithConnector))
+	h.GET("/webapi/ping", h.WithUnauthenticatedLimiter(h.ping))
```

Comment:
I'm still a little nervous about ping. `tsh login` hits this endpoint, so with the current settings, no more than 40 users can log in (from the same IP) during a 1 minute window. 

There are several ways (SNAT, VPN gateway, etc) that it could appear that all users are connecting from the same IP. 

What do you think about either using more generous limits for this endpoint, or starting with some metrics/logging and otherwise leaving it unlimited so that we can evaluate the impact of such a change?

Reply:
> What do you think about either using more generous limits for this endpoint, or starting with some metrics/logging and otherwise leaving it unlimited so that we can evaluate the impact of such a change?

I believe this is one of the more important endpoints to rate limit.  What rate limit would make you comfortable?  We could potentially keep the average at 20, but increase the burst to 100 or 120.

The goal in having this land right after v13 is cut is so that we can have as long as possible period to test and witness any unexpected interactions from this change.

Let me know your thoughts, thank you!

Reply:
I don't know what limits would make me more comfortable, because I don't have any data to inform me as to what's reasonable and expected and what's abnormal.

If we put a prometheus metric in for requests/s to this endpoint (grouped by remote IP) we wouldn't have to collect data very long to get a good idea of what's normal.

Reply:
@zmb3 and @Joerger Can you please review the latest commit: [3d31f86](https://github.com/gravitational/teleport/pull/24623/commits/3d31f86f24f890b6cf21d0c563b20362ea62c1da)

There I introduce a `high` rate limiter for these select endpoints which are only CPU bound.  After discussions with @zmb3 I believe this to be the safest balance, but please give me your candid thoughts.  Thank you!